### PR TITLE
User guide fixes

### PR DESCRIPTION
**Pull request by joefarned**
Docs quick build now works. The fix was to change "developers" to "developer" in common_conf.py. This also contains some minor bug fixes, updates pipelines label names, and removes table of contents that were not needed.

### CODE REVIEW

File: cdap-docs/user-guide/source/data-preparation/functions/table-of-contents.rst [hunk 1]
```diff
@@ -1,18 +0,0 @@
-.. meta::
```
Comment (bdmogal) [inline cdap-docs/user-guide/source/data-preparation/functions/table-of-contents.rst:1]: this is not required?

File: cdap-docs/user-guide/source/data-preparation/directives/table-of-contents.rst [hunk 2]
```diff
@@ -1,82 +0,0 @@
-.. meta::
```
Comment (bdmogal) [inline cdap-docs/user-guide/source/data-preparation/directives/table-of-contents.rst:1]: this file is not required?

File: cdap-docs/user-guide/source/data-preparation/service/table-of-contents.rst [hunk 3]
```diff
@@ -1,20 +0,0 @@
-.. meta::
```
Comment (bdmogal) [inline cdap-docs/user-guide/source/data-preparation/service/table-of-contents.rst:1]: ditto

File: cdap-docs/user-guide/source/data-preparation/table-of-contents.rst [hunk 4]
```diff
@@ -1,18 +0,0 @@
-.. meta::
```
Comment (bdmogal) [inline cdap-docs/user-guide/source/data-preparation/table-of-contents.rst:1]: ditto

File: cdap-docs/user-guide/source/pipelines/plugins/actions/index.rst [hunk 5]
```diff
@@ -18,5 +18,5 @@ These plugins work with:
     :maxdepth: 1
     :glob:
 
-    **
+    *
```
Comment (bdmogal) [inline cdap-docs/user-guide/source/pipelines/plugins/actions/index.rst:21]: why is this required?

File: cdap-docs/user-guide/source/pipelines/plugins/analytics/index.rst [hunk 6]
```diff
@@ -19,4 +19,4 @@ These plugins work with:
     :maxdepth: 1
     :glob:
 
-    **
+    *
```
Comment (bdmogal) [inline cdap-docs/user-guide/source/pipelines/plugins/analytics/index.rst:22]: ditto

File: cdap-docs/user-guide/source/pipelines/plugins/post-run-plugins/index.rst [hunk 7]
```diff
@@ -19,4 +19,4 @@ These plugins work with:
     :maxdepth: 1
     :glob:
 
-    **
+    *
```
Comment (bdmogal) [inline cdap-docs/user-guide/source/pipelines/plugins/post-run-plugins/index.rst:22]: ditto

File: cdap-docs/user-guide/source/table-of-contents.rst [hunk 8]
```diff
@@ -1,18 +1,16 @@
...
-	:titlesonly:
+   :titlesonly:
 
+    Introduction <index>
     Overview <overview>
-	Tutorials <tutorials/index>
+    Tutorials <tutorials/index>
     Data Preparation <data-preparation/index>
-    Data Pipelines <pipelines/index>
-    FAQ <faq/index>
```
Comment (bdmogal) [inline cdap-docs/user-guide/source/table-of-contents.rst:16]: how come FAQs is gone?

File: cdap-docs/user-guide/source/table-of-contents.rst [hunk 9]
```diff
@@ -1,18 +1,16 @@
...
 .. toctree::
-	:titlesonly:
+   :titlesonly:
 
+    Introduction <index>
     Overview <overview>
-	Tutorials <tutorials/index>
+    Tutorials <tutorials/index>
     Data Preparation <data-preparation/index>
-    Data Pipelines <pipelines/index>
```
Comment (bdmogal) [inline cdap-docs/user-guide/source/table-of-contents.rst:15]: let's keep this as Data Pipelines?

**Review by bdmogal — CHANGES_REQUESTED** [review]
Overall, good stuff finding these errors. Some minor questions.

File: cdap-docs/user-guide/source/data-preparation/functions/table-of-contents.rst (hunk 1 above)
Comment (joefarned) [reply cdap-docs/user-guide/source/data-preparation/functions/table-of-contents.rst:1]: The TOC tree is included in the index for the subfolders (following structure left by JJ before), so including a table-of-contents will cause a circular reference error.

File: cdap-docs/user-guide/source/pipelines/plugins/actions/index.rst (hunk 5 above)
Comment (joefarned) [reply cdap-docs/user-guide/source/pipelines/plugins/actions/index.rst:21]: This is pattern taken from JJ's work which auto-populates TOC with files in dir.

**Comment by bdmogal** [comment]
This comment needs to be addressed - https://github.com/caskdata/cdap/pull/9260#discussion_r127862799. Other than that, please post a link to the build of this branch. If that succeeds, LGTM

**Comment by joefarned** [comment]
Here's a successful build: https://builds.cask.co/browse/CDAP-DBT26-10

@bdmogal, sorry about that -- made a comment on #9260 but it is pending for some reason. I have not yet written FAQs. If we push this out now, I can write those later today and submit another PR.
